modules/ROOT/pages/design-create-publish-api-specs-from-file.adoc

```diff
@@ -0,0 +1,51 @@
+= Create and Publish an API Specification or Fragment from an External File
+
+Use API Designer to create a new API specification or an API fragment from an existing external file. API Designer supports importing from JSON, RAML, or YML files. You can also import API specifications and fragments from an existing project.
+
+== Procedure
+
+. On the *Projects* page of Design Center, click the *Create new* button and then click *Import from File*.
+. Specify a name to give to the imported project.
+. Click Choose file, then browse your filesystem for the you want to import.
++
+You can create a new project from one following file types:
++
```

Comment:
Typically, when just referring to the file type, we use JSON, RAML, YAML, and ZIP.

Reply:
Fixed.